src/components/form/form.style.ts

```diff
@@ -76,7 +85,13 @@ const formBottomMargins = (fieldSpacing) =>
 // Accounts for height of the header of Modal parent, the height form footer and some additional spacing
 const HEIGHT_SPACING = 216;
 
-export const StyledForm = styled.form`
+interface StyledFormProps extends StyledFormContentProps {
+  height?: string;
+  fieldSpacing: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7;
```

Comment:
Technically the `StyledForm` doesn't support the value `6` for `fieldSpacing` here. What do you think about adding support for this in? I would assume this would make the bottom margin `var(--spacing600)`?

Reply:
Good spot! So it's actually the `Form` component itself that doesn't allow 6, and it looks like a mistake of mine here to include the `6` option in the `StyledFormProps` when there isn't a `formBottomMargin` case for it and it technically shouldn't ever be passed from the parent anyway.

However I don't see any downside to adding 6 as an option with `var(--spacing600)` as the margin value as you say so I will get this added 👍